torch/csrc/jit/passes/utils/subgraph_utils.cpp

```diff
@@ -52,6 +52,12 @@ struct ValueMapper {
         subgraph_num_outputs_, num_outputs - subgraph_num_outputs_);
     for (Value* v : new_outputs) {
       auto maybe_last_use = firstOrLastUse(v, /*find_first*/ false);
+      if (!maybe_last_use) {
```

Comment:
i think so, it may also because we're fusing in the wrong order and i'm doing this:
```
    // TODO: subgraph utils doesnt handle merging a consumer into a producer
    // so we merge the producer into the consumer even though it's inefficient
    consumer = SubgraphUtils::createSingletonSubgraphAndUpdateAliasing(
```

I'm going to try to address that in a follow up PR. It makes the fuser slower, but this is run once AOT once so being a little slower is not a blocking issue IMO

Reply:
this does get addressed in a later PR